tests/unit/test_parser.py

```diff
@@ -14,6 +15,16 @@
         parameters.append([_file, network_os])
 
 
+# TODO: add more tests with different patterns
```

Comment:
Instead of manually generating these, see the pattern I provided in #141

Reply:
I am not sure what you mean here.

Reply:
Addressed see below.